app/templates/buyers/start_brief_info.html

```diff
@@ -40,10 +40,24 @@
     <div class="column-two-thirds large-paragraph">
         <div class="marketplace-paragraph">
             <p>
+                 {% set digital_specialists_intro %}
+                        You can use the Digital Marketplace to find a specialist, such as a developer or a user researcher, for a specific piece of work.
+                    </p> <p>
```

Comment:
Having half sets of tags in the `set` blocks and the other half outside feels a bit weird. I think it would be nicer to define all three messages as complete blocks including the paragraph tags.

e.g. 
```
{% set digital_specialists_intro %}
<p> ... </p>
<p> ... </p>
{% endset %}

{% set digital_outcomes_intro %}
<p> ... </p>
<p> ... </p>
{% endset %}

{% set user-research-participants_intro %}
<p> ... </p>
{% endset %}
```

Reply:
I suggest grabbing a nearby developer to help fix this :)

Reply:
Thanks Kev! Pea has helped me with this :-)